### DISCUSSION TITLE
How can I give hints about my vault and note structure to improve results?

### DISCUSSION THREAD

**Discussion by upnix** [brianpetro/obsidian-smart-connections#715] [General]
Examples:  
* I create daily notes titled "YYYY-MM-DD", but when I ask anything involving "last week" or "in the last month" I don't often get these notes included as context.
* I take meeting notes in this daily note. The first bullet will have a meeting summary and the line ends with the tag `#Meeting`, then all my notes for the meeting are a sub-bullets under that. I'd like to be able to ask "During my meeting with Joe, how many widgets did he say he needed?" This time the AI (rather than the embedding search) has no idea that sub-bullets of a parent bullet with "Joe" and `#Meeting` is going to have the information needed.

Using existing functionality, is there a way I can give hints to Smart Connections about the structure of my information? Some of this structure would be universal, too, like if I ask about highlighted text that's going to mean I'm looking for words between two `==`s.

Thoughts or ideas about this?

**Comment by hittjwX** [comment]
AI, in this context, isn't very good with chronology. I'm unable to get anything useful asking for a list of what I've written about recently either. Maybe SC needs to predigest notes based on `creation` and `modified` dates. 

Otherwise, I need to use [specific context with wiki links](https://github.com/brianpetro/obsidian-smart-connections/discussions/666#discussioncomment-9849584).

As a newsletter publisher, I'd love to use a prompt "Write a list of key ideas and concepts I've taken notes about over the last two weeks. Include a statement about the high level idea in terms of benefits to the reader. Link to the note please."
